meetings/ui/pom.xml

```diff
@@ -14,10 +14,8 @@
     <packaging>war</packaging>
 
     <properties>
-        <frontend-maven-plugin.version>1.12.1</frontend-maven-plugin.version>
         <frontend-maven-plugin.npmVersion>9.5.1</frontend-maven-plugin.npmVersion>
```

Comment:
I'd think this npmVersion and nodeVersion should probably be defined in master for everything that uses this plugin? But it seems to work fine with these versions here using the updated frontend plugin. So maybe they're independent?

Reply:
They are independent, I'm happy to use the same version of the frontend-maven-plugin for everything, thanks Matt @jonespm !